src/styles/sidebar/components/toast-messages.scss

```diff
@@ -0,0 +1,83 @@
+@use "../../mixins/panel";
+@use "../../variables" as var;
+
+.toast-messages {
+  position: absolute;
+  z-index: 1;
+  width: 100%;
+  left: 0;
+  padding: 0 8px;
+}
+
+.toast-message-container {
+  position: relative;
+  width: 100%;
+  animation: slidein 0.3s forwards ease-in-out, fadeout 0.5s 4.5s;
```

Comment:
If we keep both animations, I just realized that `forward` value should be applied to both animations. You can see the issue more clearly by changing the delay of 4.5s to 4.4s. The toast message blinks back for that split second.

add `animation-fill-mode: forwards;` on the next line and it should apply that to both animations.